src/Sqs/Input/SendMessageRequest.php

```diff
@@ -193,7 +191,6 @@ public function requestBody(): string
                         if (null !== $v = $input->getStringValue()) {
                             $payload["MessageSystemAttribute.{$indices->k6857220}.Value.StringValue"] = $v;
                         }
-
```

Comment:
I added some space here before. It made it a little easier to read..

What we should do is to add a comment here os something. Like 

```
// $input->getBinartValue()
```

That will help with readability. But that is another PR.